easybuild/easyconfigs/s/SPOOLES/spooles-2.2-makeInc.patch

```diff
@@ -0,0 +1,79 @@
+--- Make.inc.orig	2014-10-14 16:48:12.744906000 +1300
```

Comment:
any lines above the `---` lines are comments

see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/c/Clang/Clang-3.4.1-pic-crt.patch

we should be more rigorous to get people to include comments in their patch files...